content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -253,6 +258,18 @@ mkdir -p /etc/systemd/system/kubelet.service.d
 curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
 ```
 
+Add /opt/bin to PATH:
+
+```bash
+sed -i '$aexport PATH=$PATH:/opt/bin' /etc/profile&&source /etc/profile
+```
+
+Or,add /opt/bin to secure_path if you want to run the commands listed in /opt/bin with sudo:
+
+```bash
+sudo sed -i 's/secure_path.*/&\:\/opt\/bin/g' /etc/sudoers
+```
```

Comment:
@skycastlelily2 
maybe we shouldn't install in /opt/bin any more, instead?

/sig cluster-lifecycle
/assign @detiber @randomvariable 
hi, could you please take a look at this problem/ fix.

Reply:
Ah,we can put the commands to /usr/local/bin,which is writeable and in the default PATH and secure_path

Reply:
/usr/local/bin seems like the better location for these binaries, but i'd like to see feedback from others too.

Reply:
+1 to using /usr/local/bin

Reply:
/usr is locked in its entirety by rpm-ostree in Fedora CoreOS, isn't it? Not 100% sure though.

Reply:
i remember something like that.
actually, yes https://rpm-ostree.readthedocs.io/en/latest/manual/administrator-handbook/

> /usr has a read-only bind mount at all times. Any data in /var is never touched, and is shared across upgrades.

so where should one install their preferred binaries?

/hold

Reply:
There's some guidance here, https://ostree.readthedocs.io/en/latest/manual/adapting-existing/, maybe /var/usrlocal/bin? Don't feel confident in making a determination though.

Reply:
Might be good to see if Fedora CoreOS already supports the UserMove mentioned in that doc, if so, we might be able to just use /usr/local/bin and it'll be handled in the background by the OS

Reply:
got confirmation that /usr is locked on Fedora CoreOS but not on regular Fedora.

Reply:
It looks like the UserMove feature does some mounting logic to allow for /usr/local to be mirrored from /var/usrlocal. I can't find any docs to state if it is supported in Fedora CoreOS, though, only mainline Fedora as of version 17.